src/api.ts

```diff
@@ -227,13 +228,11 @@ router.post(
       const message: TypedMessage<BidPlacedV1Data> = {
         event: MessageType.BidPlaced,
         version: "1.0",
-        timestamp: new Date().getTime(),
+        timestamp: dateNow.getTime(),
```

Comment:
This is still the same issue I mentioned before but now it's assigned to a variable `dateNow`, because you're calling `getTime()` more then once, so it could be different. Store the value of that function instead.

`const time = new Date().getTime()`

Reply:
Sorry, used to working in UTC time, not epochs